bolt/tools/merge-fdata/merge-fdata.cpp

```diff
@@ -297,6 +298,25 @@ void mergeLegacyProfiles(const SmallVectorImpl<std::string> &Filenames) {
               "cannot mix profile collected in BOLT and non-BOLT deployments");
         BoltedCollection = false;
       }
+      // Check if the string "no_lbr" is in the first line
+      size_t FirstNewline = Buf.find('\n');
```

Comment:
Can it be on a second line, in case of boltedcollection?

Reply:
If it is, we've already dropped the first line just above when checking for boltedcollection.

Reply:
I've updated both comment and variable name to avoid confusion.